tests/test_wave_01.py

```diff
@@ -31,13 +31,15 @@
     'Z': 1
 }
 
+#@pytest.mark.skip()
```

Comment:
Leaving this as a comment is fine, but in industry, I would expect commented lines like this to be deleted before committed and pushed.